.ci/monorepo_functions.sh

```diff
@@ -22,6 +22,7 @@ get_all_packages() {
         form-types-bundle \
         migrations \
         monorepo-tools \
+        php-platform \
```

Comment:
I feel like `php-platform` name is not clear, at first it feels like some PHP-based platform project.

`php-fpm-docker-image` for example is much more apparent to me

Reply:
It was a struggle to come up with a nice name, tbh.

I had the following criteria:
- It should not be similar to the existing official php image, as the versioning will be completely different 
- It should describe that it's not just another php-fpm image, but contains more specific stuff for the desired usage
- It should be short and easy to remember
- It should at least loosely remind the Shopsys Platform
- Preferably it should not contain any unnecessary technical terms (such as fpm) so we can use the best technology at the time without worrying about the name

So the `shopsys/php-platform` seemed like the best choice.

Reply:
Okay, but I miss info about it being a Docker image. So maybe something like `platform-php-docker` or `plaform-php-docker-image` so when you read it out loud it is `Shopsys Platform PHP Docker Image`? :)

Reply:
changed to php-image as we discussed